docs/docs/bootstrapping.md

```diff
@@ -14,20 +14,30 @@ option                                        | description
 --where WHERE_CLAUSE                          | where clause to restrict the rows bootstrapped from the specified table
 --client_id CLIENT_ID                         | specify which maxwell instance should perform the bootstrap operation
 
-### Using the maxwell.bootstrap table
+### Starting a table bootstrap
 ***
-Alternatively you can insert a row in the `maxwell.bootstrap` table to trigger a bootstrap.
+You can start a bootstrap using:
 
 ```
-mysql> insert into maxwell.bootstrap (database_name, table_name) values ('fooDB', 'barTable');
+bin/maxwell-bootstrap --config localhost.properties --database fooDB --table barTable --log_level info
```

Comment:
I'm not sure why we thought you needed to specify `--config` and `--log_level` in these examples.  As long as we're updating the docs let's simplify this down to just what you need, ie `--database` and `--table`

Reply:
Done